content/post/22-04-24-progress-report-33.md

```diff
@@ -0,0 +1,283 @@
+
+The Syndicate have given their covert agents a new tool in the form of the Hypopen, a pen that doubles as a hypodermic syringe and can be used to discreetly administer drugs to unsuspecting station personnel. Additional chemical bottles can be purchased from a Syndicate Uplink that will stun injected victims.
+
+Holoparasites have been significantly improved, and are now immune to most firearms and tasers.
+
+TODO: Pic?
+
+{{< newsection >}}
+### Lizards
+*Contributed by  metalgearsloth, Daemon, Macoron, DogZeroX, moony*
+
+We've shown them off before, but the classic SS13 lizards are now officially playable in SS14 thanks to a bunch of work & fixes by several contributors. Lizard visuals and their unique accent are taken primarily from /tg/station 13.
+
+TODO: Pic of janitor lizard showing off accent.
+
+{{< newsection >}}
+### New Plushies
+*Contributed by DogZeroX and ZeroDayDaemon*
+
+Four adorable new plushies have been added: Ratvar, Rouny, RGBee. and a space lizard.
+
+TODO: Pic
+
+{{< newsection >}}
+### New Admin Features & Custom Resource Uploads
+*Contributed by Vera*
+
+Admins now have a variety of new tools to ~~abuse players~~ enhance the round, and a few minor changes have been made to the server rules. Here's the highlights:
+- Admins can now play sounds to every player or specific players.
+- Admins can now upload custom resources to the server, like new sprites.
+
+{{< video-audio "/video/pr_33/admin_abuse.mp4" >}}
+
+{{< newsection >}}
+## Mapping
+This progress report sees the addition of 3 new maps. There's also a pile of updates to all the other maps, but we don't have the time or space to cover them all here. As always, the most up-to-date images for maps can be found on our [wiki](https://wiki.spacestation14.io/wiki/Maps).
+
+### Delta Station
+*Contributed by Timrod*
+
+Delta Station, one of the largest maps in SS13, has been ported to SS14. It's so large that it's bigger than any other two SS14 maps combined. This incarnation is a mixture of /tg/'s map, Citadel's map, and a few unique changes to better fit SS14 mechanics like a remapped security department and removal of the AI satellite.
+
+{{< imgw "images/post/pr_33/delta" >}}
+
+{{< newsection >}}
+### Marathon Station
+*Contributed by Emisse*
+
+Marathon Station has been added as a heavily-modified and rechristened port of SS13's Pubby Station. Hopefully the handful of Pubby fans will feel right at home, once they figure out where some of the departments went.
+
+{{< imgw "images/post/pr_33/marathon" >}}
+
+{{< newsection >}}
+### Split Station
+*Contributed by Peptide90*
+
+To show you the power of SS14, Peptide sawed this ~~boat~~ station in half!
+
+Split Station is, as the name implies, split into two separate asteroids with a couple hallways connecting them. I sure hope nobody disconnects them and flies off with half of the station!
```

Comment:
Note the fact that you can do this, as at the time of release of this PR grid splitting is merged.